Text/PresidioPIIAnonymization/README.md

```diff
@@ -0,0 +1,215 @@
+# Text PII Anonymization Power Skill
+
+[Presidio](https://github.com/microsoft/presidio) is an open-source tool to recognize, analyze and anonymize personally identifiable information (PII). Using trained ML models, Presidio was built to ensure sensitive text is properly managed and governed.
+
+This Power Skill uses [Presidio](https://github.com/microsoft/presidio) Analyzer and Anonymizer
+to find and remove PII entities. Even though Presidio supports several anonymization methods (hash, encrypt, redact, replace, mask), 
+this Power Skill only uses redact and removes the PIIs completely from the text  .
+
+This skill is ideal for finding and removing PII entities from the search text.
+
+## Requirements
+
+In addition to the common requirements described in the root [README.md](https://github.com/Azure-Samples/azure-search-power-skills/blob/master/README.md) 
+file, this Power Skill requires spacy en_core_web_lg module being downloaded:
+```python
+python -m spacy download en_core_web_lg
+```
+
+To run this PowerSkill you will need:
+* docker
+* An Azure Blob storage container
+* A provisioned Azure Cognitive Search (ACS) instance 
+* A provisioned Azure Container Registry
+* A Cognitive Services key in the region you deploy ACS to
+
+Below is a full working example that you can get working E2E on sample data.
+
+## How to implement
+
+This section describes how to get this working on sample data and how it can be amended for your data.
+ 
+1) ### Data
+   The first step is to view the sample data files here [train data](data/). 
+1) ### Run the API
```

Comment:
Consider splitting into Install and then Run

Reply:
I followed the summarization way of putting it down.